src/pos/client.ts

```diff
@@ -281,6 +288,16 @@ export class PoolProtocolImpl extends AbstractProtocolClient {
                     unstake
                 )} CTSI, withdraw ${formatCTSI(withdraw)} CTSI`
             );
+
+            if (this.lastRebalanceTimestamp + this.rebalanceIntervalMillis > Date.now()) {
+                log.info(
+                    `[${pool.address}] not enough time from last rebalance (${
+                        this.lastRebalanceTimestamp
+                    }). rebalanceInterval = ${this.rebalanceIntervalMillis}`
```

Comment:
It would be nice to log the dates formatted, or use the [humanize-duration](https://github.com/EvanHahn/HumanizeDuration.js) lib that we already use in the project to print something like `next rebalance delayed to be executed in 6 hours and 40 minutes`

Reply:
Done